docs/content/querying/lookups.md

```diff
@@ -27,6 +27,13 @@ and such data belongs in the raw denormalized data for use in Druid.
 Very small lookups (count of keys on the order of a few dozen to a few hundred) can be passed at query time as a "map" 
 lookup as per [dimension specs](../querying/dimensionspecs.html).
 
+## Configuration
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.lookup.snapshotWorkingDir`| Working path used to store snapshot of current lookup configuration, leaving this property null will disable snapshot/bootstrap utility|null|
+|`druid.lookup.snapshotPeriod`| Periodic snapshot duration, setting to 0 will limit snapshot to only once upon druid process shutdown gracefully. |"PT10M" (10 minutes)|
```

Comment:
it is to aggressive in my opinion, i don't expect lookup config will change that much. If you are really string about this i can change it.